packages/teleport-plugin-vue-base-component/src/index.ts

```diff
@@ -31,12 +31,27 @@ export const createPlugin: ComponentPluginFactory<VueComponentConfig> = (config)
     const dataObject: Record<string, any> = {}
     const methodsObject: Record<string, any> = {}
 
-    let templateContent = createHTMLTemplateSyntax(uidl.node, {
-      templateLookup,
-      dependencies,
-      dataObject,
-      methodsObject,
-    })
+    let templateContent = createHTMLTemplateSyntax(
+      uidl.node,
+      {
+        templateLookup,
+        dependencies,
+        dataObject,
+        methodsObject,
+      },
+      {
+        interpolation: (value) => `{{ ${value} }}`,
+        eventBinding: (value) => `@${value}`,
+        valueBinding: (value) => `:${value}`,
+        eventEmmitter: (value) => `this.$emit('${value}')`,
+        conditionalAttr: 'v-if',
+        repeatAttr: 'v-for',
+        repeatIterator: (iteratorName, iteratedCollection, useIndex) => {
+          const iterator = useIndex ? `(${iteratorName}, index)` : iteratorName
+          return `${iterator} in ${iteratedCollection}`
+        },
```

Comment:
How about moving to a separate file like `DEFAULT_TEMPLATE_SYNTAX` to make this look short and crisp

Reply:
I wanted to have this visible in the main plugin, it's not a constant necessarily, as it has functions as well, it's the configuration of the template syntax function